www/source/partials/docs/_bp-robust-supervisor-networks.md.erb

```diff
@@ -0,0 +1,91 @@
+
+If your permanent peer Supervisors are not running any services, they will be less subject to the pressures that would cause service-running Supervisors to come and go. They can exist solely to anchor the entire Supervisor network.
+
+## Pulling It All Together: A Robust Supervisor Network
+
+With all this, we can come up with a robust Habitat network architecture. In fact, this is the same architecture the Habitat team uses to run the public [Builder service](https://bldr.habitat.sh).
+
+### Create the Bastion Ring
+
+First, set up three Supervisors as permanent peers, all mutually peered to each other (The labels "A", "B", and "C" are stand-ins for the IP addresses at which these Supervisors are reachable):
+
+``` sh
+# Supervisor "A"
+hab sup run --permanent-peer --peer=B --peer=C
+
+# Supervisor "B"
+hab sup run --permanent-peer --peer=A --peer=C
+
+# Supervisor "C"
+hab sup run --permanent-peer --peer=A --peer=B
+```
+
+These Supervisors should never be used to run services. They _can_ however, serve as convenient entry points to the network for doing things like injecting configurations using `hab config apply`, adding files using `hab file upload`, or departing Supervisors using `hab sup depart`.
+
+### Peer additional Supervisors to the Bastion Supervisors
+
+Each additional Supervisor you add to the network should be peered to _at least one_ of the bastion ring Supervisors. Technically speaking, only one peer is necessary, as that provides access to the rest of the network. However, it could be possible to not fully connect to all of them if, say, the Supervisor joined _during_ a network split event. Out of convention and redundancy, we peer to _all_ the bastion ring Supervisors, like so:
+
+```
+# Supervisor "D" (a "normal" Supervisor)
+hab sup run --peer=A --peer=B --peer=C
+```
+
+This Supervisor _should_ be used to run services, but _should not_ be started as a permanent peer.
+
+## Conclusion
+
+Hopefully, the above discussion has given you a better idea of how Habitat's networking works, and how you can best take advantage of it to provide a robust network foundation for the services you run.
+
+For those that like to keep things succinct, the above advice can be summed up thusly:
+1. Run three mutually-peered, permanent Supervisors
```

Comment:
A line break here would make ordered-list rendering work. (Currently the list items are all mashed into a single wrapping line.)